tigerpath/majors_and_certificates/majors/AAS_2018.json

```diff
@@ -0,0 +1,815 @@
+{
+  "type": "Major",
+  "name": "African American Studies",
+  "code": "AAS",
+  "degree": "AB",
+  "year": 2018,
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-african-american-studies",
+    "https://aas.princeton.edu/academics/undergraduate-program/concentration-requirements"
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Eddie S. Glaude Jr.",
+      "email": "esglaude@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Naomi Murakawa",
+      "email": "murakawa@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Imani Perry",
+      "email": "iperry@princeton.edu"

```

Comment:
Aren't they no longer serving as acting DUS? The undergraduate announcement seems to say that they were acting DUS for the 2019-2020 academic year only.

Reply:
Yeah, I wasn't sure if it meant only for that academic year. It's probably better to remove them.